config/eslint/lint-paths.json

```diff
@@ -1,5 +1,12 @@
 {
-  "drupal": "docroot/themes/custom docroot/modules/custom",
-  "laravel": "resources/assets/js",
-  "magento": ""
+  "js": {
+    "drupal": "docroot/themes/custom docroot/modules/custom",
+    "laravel": "resources/assets/js",
```

Comment:
core uses `Modules/*/resources/assets/js` can this be via .stylelintrc.yml ?

Reply:
Hi @jez500,

That can be configured in package.json. We are only using this to generate lint script
so it will append to your package.json script the following:

`    "lint:js": "eslint --color resources/assets/js",
    "lint:js-fix": "eslint --color --fix resources/assets/js",`